otx/cli/builder/builder.py

```diff
@@ -301,7 +275,15 @@ def build_model_config(self, model_config_path, backbone_config_path, output_pat
 
         # Build Backbone
         backbone = build_from_cfg(backbone_config, otx_registry, None)
-        out_channels = get_backbone_out_channels(backbone)
+        if model_config.model.get("task", None) == "classification":
```

Comment:
How about converting all `string variables` to the `TaskType` class?
(i.e. "classification" --> "TaskType.CLASSIFICATION")

Then, we can represents all kinds of tasks supported by Geti and also can avoid hard-coded string format.

Reply:
Perhaps I could use something like `str(TaskType.CLASSIFICATION)` instead of `classification`. I'll fix it.